akka-docs/src/main/paradox/deploying.md

```diff
@@ -1,21 +1,29 @@
-# How can I deploy Akka?
+# Package, Deploy and Run
+ 
+* [Packaging](#packaging)
+* [Deploying](#deploying)
+    - [Rolling Updates](#rolling-updates)
+    - [Rolling Updates and Akka versions](project/rolling-update.md)
+* [Running]()
+    - [Akka Management](https://doc.akka.io/docs/akka-management/current/) - a suite of tools for bootstrapping and operating Akka Cluster applications
+    - [Akka Discovery](discovery/index.md) - an interface around various ways of locating services 
```

Comment:
Discovery is its own paradox directory with one index file, for a relatively small module.
What we could consider is:

1. Create a new `/paradox/deploy/` - it would be nice to separate the new deployment into topics - I see it expanding over the next few months, just with this ticket
2. Relocate content from `/paradox/discovery/index.md` to a new `/paradox/deploy/discovery.md` and remove the current dir/index for it

Reply:
I definitely agree with the index structure and the general idea.

Note that the directory structure does not necessarily have to match the index structure, and moving pages around require adding redirect from where the pages were before to not break potential links from the outside.

Reply:
I was wondering about that, e.g. for the removal of /paradox/additional/deploy.md to the expanded top level /paradox/deploying.md what has to be done?

Reply:
So perhaps leave discovery as is, and we could still consider a `paradox/deploy/` for the new.

Reply:
Sounds good to have a Deploy or Deployment section in the docs, but as Johan said that can be done with the ToC without moving the existing physical page.

Reply:
> for the removal of /paradox/additional/deploy.md to the expanded top level /paradox/deploying.md what has to be done for redirects

In such a case we need to add a redirect to `.htaccess` to make existing links and links from the 'you are browsing an old versions of the docs' message work. Unfortunately that's required but not sufficient: we also need to go back and fix the 'canonical' links in the old versions of the docs. So indeed might be nicer to leave the directory structure intact and just change the ToC

Reply:
So keep /paradox/additional/deploy.md ?

Reply:
Yes I think so